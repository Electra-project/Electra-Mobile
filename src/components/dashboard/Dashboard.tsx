import React from 'react';
import {View, Text} from 'react-native'
export default class Dashboard extends React.Component {

  render(){
    return (<View><Text style={{paddingTop:10, paddingBottom:10, color:'#FFFFFF'}}>Dashboard Application</Text></View>)
  }

}
