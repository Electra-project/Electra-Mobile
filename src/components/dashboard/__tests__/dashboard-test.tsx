import React from 'react'
import Dashboard from '../Dashboard'
import { shallow } from 'enzyme';


describe('Dashboard', () => {
    it('renders correctly', () => {
        let wrapper = shallow(<Dashboard />)
        expect(wrapper).toMatchSnapshot()
    })
})
