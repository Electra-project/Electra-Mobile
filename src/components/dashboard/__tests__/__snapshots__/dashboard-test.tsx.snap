// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Dashboard />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#FFFFFF",
            "paddingBottom": 10,
            "paddingTop": 10,
          }
        }
      >
        Dashboard Application
      </Text>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "accessible": true,
        "allowFontScaling": true,
        "children": "Dashboard Application",
        "ellipsizeMode": "tail",
        "style": Object {
          "color": "#FFFFFF",
          "paddingBottom": 10,
          "paddingTop": 10,
        },
      },
      "ref": null,
      "rendered": "Dashboard Application",
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#FFFFFF",
              "paddingBottom": 10,
              "paddingTop": 10,
            }
          }
        >
          Dashboard Application
        </Text>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "accessible": true,
          "allowFontScaling": true,
          "children": "Dashboard Application",
          "ellipsizeMode": "tail",
          "style": Object {
            "color": "#FFFFFF",
            "paddingBottom": 10,
            "paddingTop": 10,
          },
        },
        "ref": null,
        "rendered": "Dashboard Application",
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
